### PR TITLE
Add Immunization History table to eCR Viewer

### PR DESCRIPTION
**Pull request by angelathe**
# PULL REQUEST

## Summary

Adds Immunization History table to Clinical Info section of the eCR Viewer.

- Update liquid templates to grab necessary immunizations info
- Add seed data with available immunization history data
- Add Immunization History table (sorted by Administration Date)
- Add snapshot tests of immunizations table
- Fixes formatting of Active Problems table (in Clinical Info section)

## Related Issue

Fixes Github Issue https://github.com/CDCgov/phdi/issues/1111

## Screenshots

immunization_table
unavailable_immunizations

## Additional Information

- This story is part 1/2 that deals with the Immunization History table. Follow-up [[ticket](https://app.zenhub.com/workspaces/streamline-ecr-6480bf2ee530095ab41ebbe9/issues/gh/cdcgov/phdi/1286)](https://app.zenhub.com/workspaces/streamline-ecr-6480bf2ee530095ab41ebbe9/issues/gh/cdcgov/phdi/1286) will remove the “Next Due” column in place of three other columns (Dose Number, Manufacturer, & Lot Number)

### CODE REVIEW

File: containers/ecr-viewer/src/app/view-data/components/tests/Clinical.test.tsx [hunk 1]
```diff
@@ -20,7 +20,12 @@ describe("Encounter", () => {
...
-      <ClinicalInfo activeProblemsDetails={[]} vitalData={vitalData} />,
+      <ClinicalInfo
+        activeProblemsDetails={[]}
+        vitalData={vitalData}
+        reasonForVisitDetails={[]}
+        immunizationsDetails={[]}
+      />,
     ).container;
   });
   it("should match snapshot", () => {
```
Comment (emmastephenson) [inline containers/ecr-viewer/src/app/view-data/components/tests/Clinical.test.tsx:31]: In addition to these snapshot tests, could you add a test that's more like - given a FHIR bundle, we expect these components to render? I'm not sure if we already have a test suite for that set up, but if not I think we need one.

**Review by emmastephenson — COMMENTED** [review]
Mostly looks fantastic, thank you! Just one comment about testing.

**Comment by sarahtress** [comment]
@angelathe  this looks good to me for now, will wait to do a little more in depth review with the 2nd ticket!

**Review by BobanL — APPROVED** [review]
Everything looks good to me!

File: containers/ecr-viewer/src/app/view-data/components/tests/Clinical.test.tsx (hunk 1 above)
Comment (angelathe) [reply containers/ecr-viewer/src/app/view-data/components/tests/Clinical.test.tsx:34]: @emmastephenson Added!

**Review by emmastephenson — APPROVED** [review]
This looks great!! Thank you for writing such through tests!

**Pull request merged**